NExtends/Primitives/Enum.extensions.cs

```diff
@@ -100,9 +100,9 @@ static void CheckIsEnum(Type t)
 			var memInfo = type.GetMember(enumVal.ToString());
 			var attributes = memInfo[0].GetCustomAttributes(typeof(T), false);
 
-			if (attributes.Length > 0)
+			if (attributes.Count() > 0)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use '.Any()' to test whether this 'IEnumerable<System.Attribute>' is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1155)